src/python_testing/TC_IDM_4_2.py

```diff
@@ -229,13 +296,19 @@ async def test_TC_IDM_4_2(self):
                             "MaxInterval is not of uint32 type.")
 
         # Verify MaxInterval is less than or equal to MaxIntervalCeiling
-        asserts.assert_less_equal(sub_cr1_step2_max_interval_ceiling_sec, SUBSCRIPTION_MAX_INTERVAL_PUBLISHER_LIMIT_SEC,
+        asserts.assert_less_equal(sub_cr1_step2_max_interval_ceiling_sec, subscription_max_interval_publisher_limit_sec,
                                   "MaxInterval is not less than or equal to SUBSCRIPTION_MAX_INTERVAL_PUBLISHER_LIMIT_SEC")
 
         sub_cr1_step2.Shutdown()
 
         # *** Step 3 ***
-        self.print_step(3, "Setup CR2 such that it does not have access to a specific cluster. CR2 sends a subscription message to subscribe to an attribute on that cluster for which it does not have access.")
+        # Setup CR2 such that it does not have access to a specific cluster. CR2 sends a subscription
+        # message to subscribe to an attribute on that cluster for which it does not have access.
+        self.step(3)
+
+        # Setting reportInterval values for steps 3-8
```

Comment:
why are these being changed for these steps?

Reply:
For steps 3-8 and 11-13 the reportInterval values are not part of the testing per se, they just need to be there. The other steps update those values, so I'm just resetting them to "normal".

Reply:
Can you leave the min at least as-is? The mins set at the top make sense and account for ICDs.

Reply:
Updated to only set the max interval